Makefile

```diff
@@ -81,13 +81,15 @@ test-unit:
 
 .PHONY: test-integration
  test-integration:
-	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); fi; \
-	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); fi; \
+	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); else export CONTROL_KUBECONFIG=$(CONTROL_KUBECONFIG); fi; \
+	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); else export TARGET_KUBECONFIG=$(TARGET_KUBECONFIG); fi; \
 	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1); fi; \
 	if [[ -f $(PWD)/$(MACHINECLASS_V2) ]]; then export MACHINECLASS_V2=$(PWD)/$(MACHINECLASS_V2); fi; \
 	export MC_CONTAINER_IMAGE=$(MC_IMAGE); \
 	export MCM_CONTAINER_IMAGE=$(MCM_IMAGE); \
+	export IS_CONTROL_CLUSTER_SEED=$(IS_CONTROL_CLUSTER_SEED); \
  	export CONTROL_CLUSTER_NAMESPACE=$(CONTROL_NAMESPACE); \
+ 	export TARGET_RESOURCE_GROUP=$(TARGET_RESOURCE_GROUP); \
```

Comment:
declare `TARGET_RESOURCE_GROUP` and `IS_CONTROL_CLUSTER_SEED` variable here in makefile without initialization (left for user to fill) for now, as `.env` file won't have it.
later with issue https://github.com/gardener/machine-controller-manager/issues/865 pls introduce this variable as part of `.env` only